### PR TITLE
Add inline SVG icon

### PR DESCRIPTION
**Pull request by steelywing**
Inline default SVG icon to JS.

### CODE REVIEW

**Comment by steelywing** [comment]
Then we don't need to care `plyr.svg` path 😃

**Comment by steelywing** [comment]
Then create 2 version? one is bundled, will you accept?

**Comment by SamPotts** [comment]
Sure - we can create a `plyr-with-sprite.js` - can we remove the `spriteOutput` in gulp? I don't think it's needed since you can grab the sprite from the `/dist` folder and inject that. Maybe default the placeholder in the JS to be `<svg></svg>` too.

File: demo/src/js/main.js [hunk 1]
```diff
@@ -14,7 +14,6 @@
     var instances = plyr.setup({
         debug:              true,
         title:              'Video demo',
-        iconUrl:            '../dist/plyr.svg',
```
Comment (SamPotts) [inline demo/src/js/main.js]: Can we just leave the demo as-is please

**Comment by SamPotts** [comment]
Don't worry about the changes, I'll manually merge this into `develop`. I want to make a couple of changes before it goes to `master`. In future it's best to use `develop` for PRs.

**Comment by steelywing** [comment]
Done, please have a review 😄 and I think my change of "gulpfile.js" can be better, please help.

**Comment by steelywing** [comment]
I forget to commit the fixed gulpfile.js =="

**Comment by dacostafilipe** [comment]
This is a helpful feature.

Any plans on adding it to `develop`?

**Comment by sampotts** [comment]
There's plans once I seperate out controls into a module. Then I'll inline each SVG rather than the sprite as that makes more sense to me. For now this'll have to wait. If it's a big issue, feel free to use the default CDN implementation.

**Comment by sampotts** [comment]
Closing this one for now but will re-visit later as it should be easier in the new build setup.

**Pull request closed without merging**